### PR TITLE
Integrate Jupyter e2e test into CI

### LINKED ISSUE #2174: Jupyter CR E2E test
Context: http://bit.ly/kf_jupyter_design_doc

We will need an E2E test for our the Jupyter custom resource.

### PR DESCRIPTION
**Pull request by jlewi**
Fix #2174

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2333)

### CODE REVIEW

**Comment by jlewi** [comment]
Test is passing so we just need to wait for #2323 to be submitted.

File: testing/workflows/components/kfctl_test.jsonnet [hunk 1]
```diff
@@ -172,7 +173,8 @@ local dagTemplates = [
                             ["/usr/local/bin/checkout.sh", srcRootDir],
                             env_vars=[{
                               name: "EXTRA_REPOS",
-                              value: "kubeflow/tf-operator@HEAD;kubeflow/testing@HEAD",
+                              // TODO(jlewi): Change kubeflow/testing@HEAD once #289 is submitted.
+                              value: "kubeflow/tf-operator@HEAD;kubeflow/testing@HEAD:289",
```
Comment (lluunn) [inline testing/workflows/components/kfctl_test.jsonnet]: That PR is merged, should change back?

**Pull request merged**